tests/is-winner/pass_afonso.js

```diff
@@ -0,0 +1,42 @@
+const isWinner = (candidate) =>
+  winners()
+    .getWinner(candidate)
+
+    .catch((err) => Promise.reject(candidate + ' never was a winner'))
```

Comment:
In JS, you should never reject something other than an `Error`, here you would not want to reject, since this is not an error but the return value you want.